docs/docs.go

```diff
@@ -150,6 +246,35 @@ const docTemplate = `{
         }
     },
     "definitions": {
+        "model.Country": {
+            "type": "object",
+            "properties": {
+                "cities": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "code": {
+                    "type": "string"
+                },
+                "created_at": {
```

Comment:
change to camelCase `created_at` -> `createdAt`
Apply cho mấy cái ở dưới luôn nhé